x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityInfoTransportActionTests.java

```diff
@@ -226,8 +227,13 @@ public void testUsage() throws Exception {
                 // FIPS 140
                 assertThat(source.getValue("fips_140.enabled"), is(fips140Enabled));
             } else {
+                if (explicitlyDisabled) {
```

Comment:
I don't follow how this works. We don't seem to configure the value `security.enabled` based on `explicitlyDisabled`.

Reply:
Oops you're right, I need to explicitly set the settings as well since SecurityUsageTransportAction checks it as well.